app/models/identifier.js

```diff
@@ -0,0 +1,5 @@
+import { attr, belongsTo } from '@ember-data/model';
```

Comment:
```suggestion
import Model, { attr, belongsTo } from '@ember-data/model';
```
Missing model import